app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,51 @@
+class PostsController < ApplicationController
+
+	def index
+		@posts = Post.all.order(:created_at).reverse
+	end
+
+	def new
+		@post = Post.new
+	end
+
+	def create
+		@post = Post.new(post_params)
+		if @post.save
+			flash[:notice] = "New Post Created"
+			redirect_to posts_path
+		else
+			flash[:alert] = "Error! text field can't be blank"
+			redirect_to new_post_path
+		end
+		
+	end
+
+	def show
+		@post = Post.find(params[:id])
+		@comments = @post.comments.where(params[:id]== :post_id)
```

Comment:
Don't think what you've passed into `.where` is going to work. What condition are you trying to check?